### PR TITLE
Fix access control in QGSI server - no more working

### PR DESCRIPTION
**Pull request by sbrunner**
- Fix published image build
- Better logger call
- More robust initialization on failure
- More robust disable all layer on initialization failure
- Fix database URI
- Better debugging config

### CODE REVIEW

File: docker/qgisserver/geomapfish_qgisserver/accesscontrol.py [hunk 1]
```diff
@@ -45,14 +44,20 @@ def __init__(self, server_iface):
...
         try:
             config.init(os.environ.get("GEOMAPFISH_CONFIG", "/etc/qgisserver/geomapfish.yaml"))
 
-            c2cwsgiutils.broadcast.init(None)
+            c2cwsgiutils.broadcast.init()
 
             configure_mappers()
-            engine = sqlalchemy.create_engine(config.get("sqlalchemy_slave.url"))
+            QgsMessageLog.logMessage(
+                "Connect to the database: {}".format(config.get("sqlalchemy_slave.url")),
```
Comment (pvalsecc) [inline docker/qgisserver/geomapfish_qgisserver/accesscontrol.py]: The password is part of the URL => not a good idea to log as is

File: docker/qgisserver/geomapfish_qgisserver/accesscontrol.py [hunk 2]
```diff
@@ -45,14 +44,20 @@ def __init__(self, server_iface):
...
+                "GeoMapFishAccessControl",
+                level=Qgis.Info,
+            )
+            engine = sqlalchemy.create_engine(config["sqlalchemy_slave.url"])
```
Comment (pvalsecc) [inline docker/qgisserver/geomapfish_qgisserver/accesscontrol.py]: I recommend adding `pool_recycle=30` to every SQL engine. Now a day, idle connections are closed by the server after 120s.

Comment (pvalsecc) [reply docker/qgisserver/geomapfish_qgisserver/accesscontrol.py]: If you don't do that, you'll have errors!

File: docker/qgisserver/geomapfish_qgisserver/accesscontrol.py (hunk 1 above)
Comment (sbrunner) [reply docker/qgisserver/geomapfish_qgisserver/accesscontrol.py]: +1, thanks :-)

File: docker/qgisserver/geomapfish_qgisserver/accesscontrol.py (hunk 2 above)
Comment (sbrunner) [reply docker/qgisserver/geomapfish_qgisserver/accesscontrol.py]: done :-)

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/qgisserver/geomapfish.yaml_tmpl [hunk 3]
```diff
@@ -3,11 +3,18 @@ vars:
   schema: '{PGSCHEMA}'
   schema_static: '{PGSCHEMA_STATIC}'
   sqlalchemy.url: postgresql://{PGUSER}:{PGPASSWORD}@{PGHOST}:{PGPORT}/{PGDATABASE}
```
Comment (arnaud-morvan) [inline geoportal/c2cgeoportal_geoportal/scaffolds/create/qgisserver/geomapfish.yaml_tmpl]: It seems `sqlalchemy.url` is not used anymore

Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/scaffolds/create/qgisserver/geomapfish.yaml_tmpl]: Probably yes but I like to keep it...

**Pull request merged**